bears/rest/rstcheckBear.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class rstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file, code_block_language_ignore: str,
+            ):
+        """
+        A Bear for rstcheck (https://pypi.python.org/pypi/rstcheck)
+
+        rstcheck checks syntax of reStructuredText and code blocks
+        nested within it
+        supports bash, c, cpp, json, python and rst
```

Comment:
Is this a new line ? This will render in HTML as:
`rstcheck checks syntax of reStructuredText and code blocks nested within it supports bash, c, cpp, json, python and rst` (in 1 line)

You want to add periods and start with capitals.